net-analyzer/nuttcp/nuttcp-8.1.4.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils readme.gentoo-r1 systemd toolchain-funcs
```

Comment:
I'm currently not overseeing why `eutils` is needed anymore?

Reply:
eutils removed